src/containers/subjects-launch/SubjectWTSStarAlignment/aws.ts

```diff
@@ -0,0 +1,30 @@
+import { Auth } from '@aws-amplify/auth';
+import { InvokeCommand, LambdaClient } from '@aws-sdk/client-lambda';
+import { REGION } from '../../../config';
+
+const STAR_ALIGNMENT_LAMBDA_NAME = 'star-align-nf-batch-job-submission';
+
+export type StarAlignmentPayload = {
+  portal_id?: string;
```

Comment:
Will; could you please change this to `portal_run_id`.

@scwatts Also notice that, if we could harmonise this naming at job submission Lambda side to be called `portal_run_id` as well, please.

Thanks both..!

Reply:
No worries!